### DISCUSSION TITLE
What is the new status attribute?

### DISCUSSION THREAD

**Discussion by gabrielbergeron** [iptv-org/community#171] [Q&A] [answered]
I can see that many channels where modified to add the status attribute.

In the ca.m3u many channels have been marked with the status value error or blocked, does that mean that the channel was blocked or taken down by a dmca? What makes the bot change the value to these values?
Because these channels don't appear in the list once the playlist is generated.

I looked in the documentation and the readme but couldn't find anything on that.

Thanks!

**Comment by freearhey** [comment, marked as answer]
The status of each stream is updated daily by the [iptv-bot](https://github.com/apps/iptv-bot). 

The attribute can have one of the following values: 

- `online` - stream was successfully loaded
- `blocked` - server returned HTTP code [403 Forbidden](https://en.wikipedia.org/wiki/HTTP_403)
- `timeout` - request time exceeded 60 seconds
- `error` - stream could not be loaded for some other reason

Links to channels that require removal by rights holders are removed from playlists completely. Also, the id of such channels are added to a separate list: https://github.com/iptv-org/database/blob/master/data/blocklist.csv

**Reply by freearhey** [reply]
Attribute description added to the Contributing Guide: https://github.com/iptv-org/iptv/commit/72d125c56f6eb444b285fe31e50d16f4d1485e4d

**Reply by gabrielbergeron** [reply]
Thanks for the info!
I understand why this was implemented, but how is that supposed to work with the geo-blocked urls? For example I can see that in ca.m3u tvg-id="CFTM.ca" is marked blocked but it works on my side.

**Reply by freearhey** [reply]
The point is that the GitHub servers are used to check the links, which are mainly located in the US. It means that if the broadcast is not available on US territory, the bot will mark the stream as "blocked". At the moment, unfortunately, we can not bypass this constraint.
